react/react2/week1/usear-searcher/src/components/UseContext/UseContext.jsx

```diff
@@ -0,0 +1,35 @@
+import { createContext, useState } from "react";
+import PropTypes from "prop-types";
+// import { fetchingUsers } from "../fetchAPI";
+
+export const ContextWrapper = createContext({});
+
+const UseContext = ({ children }) => {
+  const [users, setUsers] = useState([]);
+  const [query, setQuery] = useState("");
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(null);
+
+  return (
+    <ContextWrapper.Provider
```

Comment:
maybe a bit overkill having the contextwrapper in a seperate component, but not wrong